### PR TITLE
chore: Add Makefile, add OpenApi Validation workflow

### PR DESCRIPTION
**Pull request by linuxluigi**
This pull request includes updates to the Makefile and README.md files, as well as the addition of a Postman Collection Validation workflow. The Makefile has been updated to include lint-oas and convert2openapi targets, and the README.md file now has an updated online documentation link. The new workflow validates the Postman Collection and converts it to OpenAPI format.

### CODE REVIEW

File: README.md
Comment (timrab) [inline README.md:1]: @linuxluigi shouldn't we also mention https://api-v2.exivity.com/ here for the v2 api?

Comment (linuxluigi) [reply README.md:1]: Tell me if you want to see it there 🙂

Comment (linuxluigi) [reply README.md:1]: Until now I just fixed a linting error

Comment (linuxluigi) [reply README.md:1]: Done

**Pull request merged**